### PR TITLE
More pootlefs stuff

### PR DESCRIPTION
**Pull request by unho**
- Remove no longer needed code
- Add system checks
- Changed pootle_fs_git requirement

### CODE REVIEW

File: pootle/checks.py [hunk 1]
```diff
@@ -490,3 +490,36 @@ def check_canonical_url(app_configs=None, **kwargs):
...
+
+    from django.conf import settings
+
+    missing_setting_error = checks.Critical(
+        _("POOTLE_FS_WORKING_PATH setting is not set."),
+        hint=_("Set `POOTLE_FS_WORKING_PATH` in settings"),
+        id="pootle.C019",
+    )
+    missing_directory_error = checks.Critical(
+        _("Path pointed by POOTLE_FS_WORKING_PATH doesn't exist."),
```
Comment (dwaynebailey) [inline pootle/checks.py]: pointed *to* by

File: pootle/checks.py [hunk 2]
```diff
@@ -490,3 +490,36 @@ def check_canonical_url(app_configs=None, **kwargs):
...
+        hint=_("Create the directory pointed by `POOTLE_FS_WORKING_PATH` "
+               "setting."),
+        id="pootle.C020",
+    )
+    not_writable_directory_error = checks.Critical(
+        _("Path pointed by POOTLE_FS_WORKING_PATH is not writable by Pootle."),
+        hint=_("Add write permission for the directory pointed by "
+               "`POOTLE_FS_WORKING_PATH` setting."),
```
Comment (dwaynebailey) [inline pootle/checks.py]: Add the write permissions to the `POOTLE_FS_WORKNG_PATH` directory using 'chmod +w'.

Hints are not meant to just repeat the error.

File: pootle/checks.py [hunk 3]
```diff
@@ -490,3 +490,36 @@ def check_canonical_url(app_configs=None, **kwargs):
...
+
+@checks.register()
+def check_pootle_fs_working_dir(app_configs=None, **kwargs):
+    import os
+
+    from django.conf import settings
+
+    missing_setting_error = checks.Critical(
+        _("POOTLE_FS_WORKING_PATH setting is not set."),
+        hint=_("Set `POOTLE_FS_WORKING_PATH` in settings"),
```
Comment (dwaynebailey) [inline pootle/checks.py]: Drop the hint

**Comment by unho** [comment]
Updated with requested changes.

File: pootle/checks.py [hunk 4]
```diff
@@ -490,3 +490,36 @@ def check_canonical_url(app_configs=None, **kwargs):
...
+        hint=_("Create the directory pointed by `POOTLE_FS_WORKING_PATH` "
+               "setting."),
+        id="pootle.C020",
+    )
+    not_writable_directory_error = checks.Critical(
+        _("Path pointed to by POOTLE_FS_WORKING_PATH is not writable by "
+          "Pootle."),
+        hint=_("Add the write permission to the `POOTLE_FS_WORKING_PATH` "
+               "directory using 'chmod +w'"),
+        id="pootle.C020",
```
Comment (phlax) [inline pootle/checks.py]: should the errors have different codes?

**Comment by phlax** [comment]
apart from nit - lgtm

File: pootle/checks.py (hunk 4 above)
Comment (unho) [inline pootle/checks.py]: Absolutely.

**Comment by ta2-1** [comment]
lgtm after c21 added

**Pull request merged**